contracts/v0.8/DataCapAPI.sol

```diff
@@ -75,8 +75,7 @@ library DataCapAPI {
             return (0, result.deserializeBytesBigInt());
         }
 
-        CommonTypes.BigInt memory empty_res;
```

Comment:
What is the reason for this change? Is it purely for an optimization? From a quick test in remix it seems that the following change is a large improvement to gas usage compared to the diff in this PR:

```
contract Return {
    BigInt empty_res;
    
    struct BigInt {
        bytes val;
        bool neg;
    }

    function getBigInt(uint256 exit_code) public view returns (uint256, BigInt memory) {
        return (exit_code, empty_res);
    }
}
```

Reply:
All *API.sol are libraries (not contracts), so the storage vars cannot be used. Constants of custom structs cannot be used also.